queries/lua/highlights.scm

```diff
@@ -7,8 +7,8 @@
 [
   "if" 
   "then"
-  "end"
```

Comment:
you could add `(end_keyword)` here instead of writing another line. Same for the others